### ISSUE TITLE
Click on pages / group album then upload goes to personal photo

### ISSUE THREAD

**Issue by prepperz** [PHPfox-Official/phpfox-v4-issues#2482]
Click any pages album > upload photo 

You'll notice that it does not upload to the album you've clicked or show the album in any option. Once all upload is completed it goes your personal photo.

Also you cannot move any of the photo to the pages album either.

**Comment by harrison05** [comment]
I can't reproduce this issue. This feature is working well on my site v4.7.1
![screen shot 2018-12-17 at 09 36 32](https://user-images.githubusercontent.com/41947198/50063548-7263f900-01df-11e9-8f71-fc3465e80b69.png)

**Comment by prepperz** [comment]
The point is, it's not going to the right place. For example. 

When you goto to a page > click photo > then upload to a pages album like "cars images" it works fine.

However, if you go to the page > click on photo > then click on the "cars images" album > then upload > it does not let select the "cars images" because it is not there. So you assume that because you click on the "cars images" album that it would be automatically uploaded to the "cars images" album so you don't select an album, but when it's uploaded it doesn't go to "cars images" album, but to your personal photo that has nothing to do with pages.

**Comment by harrison05** [comment]
Do you click on "Upload Photos to Album" action? I still can't reproduce as your issue, you can only upload photos to albums that you're the owner.
![screen shot 2018-12-19 at 10 28 07](https://user-images.githubusercontent.com/41947198/50197247-d79f2200-0378-11e9-825f-8f22e4520e0d.png)

**Comment by prepperz** [comment]
No. I did not click on "upload photos to album." I clicked on the green plus and it didn't show any options, but if you go and and upload it will show in your personal photos and not the pages photos.
![screenshot](https://user-images.githubusercontent.com/19594886/50199910-75ff9900-0321-11e9-9b34-d0255180ab34.png)


![1](https://user-images.githubusercontent.com/19594886/50199812-f5d93380-0320-11e9-8404-f4f2b4215d97.png)

![2](https://user-images.githubusercontent.com/19594886/50199814-fbcf1480-0320-11e9-9a8b-3f61afd2d262.png)


https://v4.phpfox.com/stupid/

**Comment by harrison05** [comment]
I understand, when you click on **green plus button** in Album detail page that means you upload photos to main section of Photo App, not upload to the parent Page of that Album, so you can't select any albums posted in Page.
